src/sentry/integrations/github/webhook.py

```diff
@@ -38,43 +38,62 @@ def get_external_id(username):
 
 
 class Webhook(object):
-    def __call__(self, event, organization=None):
+    def _handle(self, event, organization, repo):
         raise NotImplementedError
 
+    def __call__(self, event):
+        integration = Integration.objects.get(
+            external_id=event['installation']['id'],
+            provider='github',
+        )
+        # TODO(maxbittker) select org directly to make this pattern scale to many
+        # orgs on one integration
+        organizations = list(integration.organizations.all())
+
+        for org in organizations:
+
+            try:
```

Comment:
should you add a check before this for `if 'repository' in event`? because otherwise i think the installation event might blow up